packages/datadog-plugin-cypress/src/plugin.js

```diff
@@ -37,7 +40,7 @@ const CYPRESS_STATUS_TO_TEST_STATUS = {
 function getTestSpanMetadata (tracer, testName, testSuite, cypressConfig) {
   const childOf = getTestParentSpan(tracer)
 
-  const commonTags = getTestCommonTags(testName, testSuite, cypressConfig.version)
+  const commonTags = getTestCommonTags(testName, testSuite, cypressConfig.version, getTestType(TEST_FRAMEWORK_NAME))
```

Comment:
I think we can avoid importing a new function and just letting that responsibility to `getTestCommonTags`
```suggestion
  const commonTags = getTestCommonTags(testName, testSuite, cypressConfig.version, TEST_FRAMEWORK_NAME)
```
(see comments below)